### PR TITLE
Proposal operations

### PR DESCRIPTION
**Pull request by alexplevako**

### CODE REVIEW

File: libraries/chain/evaluators/proposal_create_evaluator.cpp [hunk 1]
```diff
@@ -27,40 +28,18 @@ void proposal_create_evaluator::do_apply(const proposal_create_evaluator::operat
               "Proposal life time is not in range of ${min} - ${max} seconds.",
               ("min", SCORUM_PROPOSAL_LIFETIME_MIN_SECONDS)("max", SCORUM_PROPOSAL_LIFETIME_MAX_SECONDS));
 
-    FC_ASSERT(_committee_service.is_exists(op.creator), "Account \"${account_name}\" is not in committee.",
+    committee_i& committee_service = committee_accessor(db()).get_committee(op.operation);
```
Comment (Andrei-Masilevich) [inline libraries/chain/evaluators/proposal_create_evaluator.cpp]: Why 'accessor'? It is unsuitable name maybe.

File: libraries/chain/include/scorum/chain/schema/proposal_object.hpp [hunk 2]
```diff
@@ -11,19 +12,26 @@ class proposal_object : public object<proposal_object_type, proposal_object>
 public:
     using cref_type = std::reference_wrapper<const proposal_object>;
 
-    CHAINBASE_DEFAULT_CONSTRUCTOR(proposal_object)
+    proposal_object() = delete;
+
+    template <typename Constructor, typename Allocator>
```
Comment (abolinbsu) [inline libraries/chain/include/scorum/chain/schema/proposal_object.hpp]: Use CHAINBASE_DEFAULT_DYNAMIC_CONSTRUCTOR here

File: libraries/chain/evaluators/proposal_create_evaluator.cpp (hunk 1 above)
Comment (alexplevako) [reply libraries/chain/evaluators/proposal_create_evaluator.cpp]: agree lets find better one.

File: libraries/chain/include/scorum/chain/evaluators/evaluator.hpp [hunk 3]
```diff
@@ -36,7 +36,7 @@ class evaluator_impl : public evaluator<OperationType>
         return OperationType::template tag<typename EvaluatorType::operation_type>::value;
     }
```
Comment (abolinbsu) [inline libraries/chain/include/scorum/chain/evaluators/evaluator.hpp]: evaluator::get_type() method is never used

File: libraries/chain/include/scorum/chain/services/proposal_executor.hpp [hunk 4]
```diff
@@ -0,0 +1,40 @@
...
+struct data_service_factory_i;
+struct proposal_service_i;
+class proposal_object;
+
+struct proposal_executor_service_i
+{
+    virtual void operator()(const proposal_object& proposal) = 0;
+};
+
+class dbs_proposal_executor : public dbs_base, public proposal_executor_service_i
```
Comment (Andrei-Masilevich) [inline libraries/chain/include/scorum/chain/services/proposal_executor.hpp]: Bad solution to use dbs_base for this class. dbs-service intend to cover only db-index operations.
This class should use new factory to support interface proposal_executor_service_i. New type of objects.

**Pull request merged**